opencbm/cbmcopy/main.c

```diff
@@ -30,6 +30,8 @@
 #include "statedebug.h"
 
 
+#define FN_BUFFER_SIZE 48
```

Comment:
I like that you change the constant to a define.
Better would be that the whole string handling is rewritten, but this is a story for another, bigger PR.

Reply:
> Better would be that the whole string handling is rewritten

I agree, but I'm not sure I have enough motivation.

Reply:
Oh, that's ok. I did not expect you to do that!